frontend/src/features/admin-form/common/AdminViewFormService.ts

```diff
@@ -27,7 +27,7 @@ import {
 import { PREVIEW_MOCK_UINFIN } from '../preview/constants'
 
 // endpoint exported for testing
-export const ADMIN_FORM_ENDPOINT = 'admin/forms'
+export const ADMIN_FORM_ENDPOINT = '/admin/forms'
```

Comment:
This change is to be consistent with `PUBLIC_FORMS_ENDPOINT` in `PublicFormService.ts`